hangman.js

```diff
@@ -1 +1,76 @@
-console.log("welcome to hangman");
\ No newline at end of file
+//Welcome Message
+console.log("welcome to hangman");
+// Prompts user to get secret word --> UNCOMMENT WHEN DONE <-- secretWord = prompt("Please select a secret word!")
+
+  //Hardcoded secretWord, delete when done.
+  var secretWord = 'toast'
+  console.log('secretWord = ' + secretWord)
+
+secretWordHidden = []
+letterGuesses = []
+
+secretWordArray = secretWord.split('');
+
+
+// Function to show user the amount of letters in the secret word
+function spaces(){
+  for (i = 0; i < secretWord.length; i++) {
+    secretWordHidden.push('_');
+  }
+} spaces()
+
+//Loop to start the letter guessing protocol
+for (i = 0; (i < (secretWord.length)); i++) {
+  //Executes prompt
+  letterGuesses.push(prompt('Word:  ' + secretWordHidden.join(' ')+ '\n' + 'Please guess a letter.  Guesses left: ' + ((secretWord.length + 1)- i)))
```

Comment:
I think this last bit ```((secretWord.length + 1)- i))``` might be a little wonky. Maybe consider setting a fixed number of incorrect guesses, and then simply counting down from that?